src/lib.rs

```diff
@@ -1213,6 +1213,153 @@ pub mod contract {
 
         type E2EResult<T> = Result<T, Box<dyn std::error::Error>>;
 
+        #[ink_e2e::test]
+        async fn reversed(mut client: ink_e2e::Client<C, E>) -> E2EResult<()> {
```

Comment:
name it : `test_swap y_to_x` and similar case but in different direction `test_swap_x_to_y`